.changeset/strange-tools-confess.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/core-components': patch
+'@backstage/plugin-catalog-react': patch
+---
+
+Make `Kind` and `Type` entity filter form fields full width
```

Comment:
good changeset! this would just effect `@backstage/core-components` when you would go with removing the `maxWidth`.